client/app/widgets/QuickCreate/__tests__/QuickCreate.spec.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import axios from 'axios';
+import {
+  render,
+  screen,
+  getNodeText,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import {
+  QuickCreate,
+  sortAlpha,
+  labelExists,
+  addToCheckboxes,
+} from 'widgets/QuickCreate';
+
+const checkboxes = [
+  { id: 'last', value: 'a_value', label: 'zoo_label' },
+  { id: 'middle', value: 'middle_value', label: 'middle_label' },
+  { id: 'first', value: 'z_value', label: 'alpha_label' },
+  { id: 'dm', value: 'duplicate_middle_value', label: 'middle_label' },
+];
+
+// https://testing-library.com/docs/dom-testing-library/api-events#fireeventeventname
+// from rtl docs Most projects have a few use cases for fireEvent, but the majority of the time
+// you should probably use @testing-library/user-event.
+describe('QuickCreate', () => {
+  beforeEach(() => {
+    render(
+      <QuickCreate
+        name="name"
+        id="123"
+        label="label"
+        checkboxes={checkboxes}
+        formProps={{
+          inputs: [
+            {
+              id: 'category_name',
+              label: 'Name',
+              name: 'category[name]',
+              required: true,
+              type: 'text',
+              value: '',
+            },
+            {
+              id: 'submit',
+              name: 'submit',
+              type: 'submit',
+              value: 'Submit',
+            },
+          ],
+          action: 'https://if-me.org/quick-create',
+        }}
+      />,
+    );
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+  describe('When the component renders', () => {
+    it('renders an input with a label with the accordion closed', () => {
+      expect(screen.queryByTestId('accordion-closed')).toBeInTheDocument();
```

Comment:
I believe we are on the same page regarding the first part of the comment. 

For the second part, thanks for calling that out. I read https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence and ran off with `use queryBy` for existence checks somehow.  I do feel that it would be a simpler mental model if that was the case, but I'll fix it to be inline with the standards! Thanks!